android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -2219,7 +2219,7 @@ public void setFullscreen(boolean fullscreen) {
         WindowInsetsControllerCompat controller = new WindowInsetsControllerCompat(window, window.getDecorView());
         if (isFullscreen) {
             eventEmitter.fullscreenWillPresent();
-            if (controls && fullScreenPlayerView != null) {
```

Comment:
not sure why you did that, but in the absolute, it makes sense.
BTW, I think this "control" test is not here as if controls == false, fullScreenPlayerView shall also be null